### DISCUSSION TITLE
Clicking merge doesn't update the site.

### DISCUSSION THREAD

**Discussion by DBcosmos** [GitbookIO/community#444] [0. General bugs and questions] [answered]
I've made edits to some text on my docs and on the dashboard no errors display. All the changes remain and it's synced with GitHub. GitHub repository also reflets these changes. but when access the xxxx.gitbook.io site to view my docs the text hasn't been updated.
Am I doing something wrong?

**Comment by KangLeian** [comment]
Hi here, I'm facing the same issues.
And I found that if I did change the end point the new URL shows my updates and changes. And then when I hit the old URL it still existed and didn't disappear.

And I've found there should be a button **Generate a new link**. Found it [here](https://docs.gitbook.com/published-documentation/share/share-links#publish-with-share-links)

But on my side, it didn't show any. It has a **Copy published link**.
How do we revoke it? or is it an Issue?

@SamyPesse @spastorelli @jpreynat @taranvohra Please kindly help us.

**Reply by DBcosmos** [reply]
Hey, glad to know I'm not the only one. My edits have been updated ok the site now. Perhaps there is a delay of some sort :)

**Comment by SamyPesse** [comment, marked as answer]
The issue has been resolved, we had an issue causing a long delay in our system to purge the cache. Sorry for the inconvenience.
